docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -41,7 +41,8 @@ dnf --assumeyes install make gcc gcc-c++ glibc-devel git-core libsecret-devel rp
 #### Arch
 
 ```sh
-TODO: Arch instructions
+# Install the development packges
+pacman -S base-devel libxkbfile libsecret libx11
```

Comment:
To fix the `deb` build you also need `libxcrypt-compat`.

```suggestion
pacman -S base-devel libxkbfile libsecret libx11 libxcrypt-compat
```

Reply:
Why is this marked as resolved? Is `libxcrypt-compat` not needed to build the deb?

Reply:
It is needed. So I don't know

Reply:
@Sertonix dw, not directed at you

Reply:
@CatPerson136 this hasn't bee resolved yet!